lib/course_creation_manager.rb

```diff
@@ -63,7 +63,7 @@ def invalid_wiki?
   end
 
   def invalid_slug?
-    @slug.split('/').length <= 1
+    @slug.split('/').reject { |slug_part| slug_part.blank? }.size <= 1
```

Comment:
What this method is doing and why is not very clear. A comment and/or rewriting the code to make the concept more clear — that to be valid, a course slug must consist of some non-blank text followed by a slash followed by some more non-blank text.